docker/docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: '3.3'
+
+services:
+   db:
+     image: mysql:5.7
+     volumes:
+       - db_data:/var/lib/mysql
+     restart: always
+     environment:
+       MYSQL_ROOT_PASSWORD: m4ster_s3cret
+       MYSQL_DATABASE: libreehr
+       MYSQL_USER: libreehr
+       MYSQL_PASSWORD: s3cret
+
+   librehealth:
+     depends_on:
+       - db
+     image: librehealth:${PHP_DOCKER_VERSION}-apache
```

Comment:
Er see comments -- `librehealth:ehr-php7.0-apache`  -- there is little reason to support multiple PHP versions :)

Reply:
Also service name should be `librehealth_ehr` or `lh-ehr`, or just `ehr`.

Reply:
Will change the service name as desired. 

Which `php` versions would you prefer: `php7.0`, php7.1`, `php7.2` (`php7.3` it's on the way). I presume the current `7.0` is Ok?

Reply:
PHP 7.x is fine -- easy to fix.

Reply:
Fixed then.